Lib/test/test_warnings/__init__.py

```diff
@@ -877,6 +877,18 @@ def test_showwarning(self):
                                 file_object, expected_file_line)
         self.assertEqual(expect, file_object.getvalue())
 
+        # bpo-35178: Test custom formatwarning can receive line as positional
```

Comment:
I would prefer to put the new test in a new method:

 def test_formatwarning_override(self):

And maybe move the new test inside test_formatwarning().

Reply:
I initially had it in `test_showwarning` since the exception was triggered by `showwarning`. I moved this into a new test `test_formatwarning_override`. Let me know if I need to move the test into `test_formatwarning` .